test-data/data/.env.example

```diff
@@ -0,0 +1,31 @@
+# Contentful APIs
+CTF_SPACE_ID="ABC123"
+CTF_CDA_ACCESS_TOKEN="ABCDEF123456"
+# CTF_CPA_ACCESS_TOKEN is the contentful preview token. You'll most likely not want to set this
+# in a .env file in order to avoid it being accessible to the nuxt client.
+# Instead set it as a ENV variable on your machine.
+CTF_CPA_ACCESS_TOKEN="GHIJKLM7891011"
+
+# Europeana APIs
+EUROPEANA_API_KEY="YOUR_API_KEY"
+EUROPEANA_ENTITY_API_KEY="YOUR_API_KEY"
+
+# HTTP Digest authentication
+# NB: on live deployments, do not set in a .env file as it may be leaked to clients.
+HTTP_DIGEST_ACL='[{"username":"user1","password":"password1"},{"username":"user2","password":"password2"}]'
+HTTP_DIGEST_REALM="demo.europeana.eu"
+
+# IP address access restriction
+IP_FILTER_ALLOW="192.168.1.0/24 127.0.0.1"
+IP_FILTER_TRUST_PROXY="10.0.0.0/8"
+
+# The Google Tag Manager ID
+# Used by @nuxtjs/google-tag-manager to allow injection of scripts via google tag manager.
+GOOGLE_TAG_MANAGER_ID="GTM-XXXXXXX"
+
+# Redirect all record page requests to this URL path
+RECORD_PAGE_REDIRECT_PATH=/record-page-under-construction
+
+# If enabled, the content tier toggle will be displayed on the main search
+# results page. Disabled by default.
+ENABLE_CONTENT_TIER_TOGGLE=1
```

Comment:
This seems like the Portal.js .env example file. How is it used in this project?